api/views.py

```diff
@@ -1182,4 +1183,13 @@ def reader_status_update(request):
 def actual_districts(request):
     rows = District.objects.all().order_by('-sort_weight', '-id').values('pk', 'title', 'is_ginekolog')
     rows = [{"id": -1, "label": "НЕ ВЫБРАН"}, *[{"id": x['pk'], "label": x["title"] if not x['is_ginekolog'] else "Гинекология: {}".format(x['title'])} for x in rows]]
-    return JsonResponse(rows, safe=False)
+
+    users = users_by_group(['Лечащий врач'])
+    users = [{"id": -1, "label": "НЕ ВЫБРАН"}, *[{'id': row[0], 'label': row[1]} for row in users]]
+
+    purposes = DoctorCall.PURPOSES
+    print(list(purposes))
+    purposes = [{"id": -1, "label": "НЕ ВЫБРАН"}, *[{'id': row[0], 'label': row[1]} for row in list(purposes)]]
```

Comment:
Можно без
```suggestion
    purposes = [{"id": -1, "label": "НЕ ВЫБРАН"}, *[{'id': row[0], 'label': row[1]} for row in purposes]]
```
Это же кортеж